src/tools/cephfs/top/cephfs-top

```diff
@@ -1112,6 +1205,9 @@ if __name__ == '__main__':
                         type=float_greater_than,
                         help='Refresh interval in seconds '
                         f'(default: {DEFAULT_REFRESH_INTERVAL})')
+    parser.add_argument('--dump', dest='dump', action='store_true',
+                        help='Dump the values to stdout')
```

Comment:
Instead of just `values`, I guess `stat/metric values`. And other places too ( doc ).